#!/bin/sh /usr/share/dpatch/dpatch-run
## 03-icons.dpatch by Daniel Baumann <daniel@debian.org>
##
## DP: Adjusts path for icons directory.

@DPATCH@

diff -Naur tinyerp-client-4.0.0-rc1.orig/bin/tinyerp-client.py tinyerp-client-4.0.0-rc1/bin/tinyerp-client.py
--- tinyerp-client-4.0.0-rc1.orig/bin/tinyerp-client.py	2006-11-27 12:30:44.000000000 +0000
+++ tinyerp-client-4.0.0-rc1/bin/tinyerp-client.py	2006-12-05 12:09:48.000000000 +0000
@@ -119,7 +119,7 @@
 if not os.path.isdir(pix_file):
 	pix_file = os.path.join(options.options['path.pixmaps'],'icons')
 if not os.path.isdir(pix_file):
-	pix_file = '/usr/share/pixmaps/tinyerp-client/icons'
+	pix_file = '/usr/share/icons'
 
 for fname in os.listdir(pix_file):
 	ffname = os.path.join(pix_file,fname)
